spec/requests/api/v3/bikes_request_spec.rb

```diff
@@ -74,6 +74,83 @@
       end
     end
 
+    context "if the bike being created already exists" do
```

Comment:
> Bike was created by a user in an organization and has been claimed - creation organization posts a matching bike (what should happen?)

Intuitive default: update the ownership 
(Could this be a situation where a bike owner has sold a bike?)

> User who was never involved with original bike posts a bike that matches a bike a different user owns - including matching that users email (what should happen?)

Intuitive default: reject the request
Alternatively: would it be useful to have some record of that attempt? (in the form of a duplicate bike record, that is)

Reply:
First one - I absolutely agree. (This would be the user returning to the shop they bought the bike from to get it serviced and the shop registers it)

Second one - I think either option could result in unintuitiveness/frustration - rejecting the request or creating a duplicate record. I think we need another way to deal with this.

Reply:
For the second case, the user story I mainly have in mind is of a malicious actor, but I'm not totally sure it applies or, if it does, that it exhausts the possibilities 🤔

Reply:
We can discuss the second case offline. I don't think it's important to deal with right now.